.github/workflows/TestStatic.yml

```diff
@@ -58,6 +61,10 @@ jobs:
         run: |
           sed -i '/^.*container_yamls.*$/i /** @phpstan-ignore-next-line */' web/sites/default/settings.php
 
+      - name: Run Security Tests
+        if: ${{ github.event_name == 'schedule' }}
+        run: ddev task test:security
```

Comment:
I wonder if we should update the name. Otherwise, reviewers will see "Run security tests" in regular PRs, and wonder why they're skipped?